.github/actions/git-check-release/action.yml

```diff
@@ -0,0 +1,18 @@
+# (C) 2022 GoodData Corporation
+
+name: "Action to update hugo version param in parmas.toml"
+description: "Action add new version to hugo version config set it latest and remove old one"
```

Comment:
fix the description & name

Reply:
this manual WF needs to be deleted it was there for testing